client/src/linkManager.js

```diff
@@ -127,6 +129,8 @@ export class linkManager extends LuigiClientBase {
    * @param {Object} [modalSettings] opens a view in a modal. Use these settings to configure the modal's title and size
    * @param {string} modalSettings.title modal title. By default, it is the node label. If there is no label, it is left empty
    * @param {('l'|'m'|'s')} [modalSettings.size="l"] size of the modal
+   * @param {string} modalSettings.width lets you specify a precise width for the modal. Allowed units are 'px', '%', 'rem' and 'em'.
+   * @param {string} modalSettings.height lets you specify a precise height for the modal. Allowed units are 'px', '%', 'rem' and 'em'.
```

Comment:
I think we also need to update the ts typings, otherwise it will be hard to use in typescript https://github.com/SAP/luigi/blob/63196f9f42e6e8ee2323a06fdb8fb3e723550d1e/client/luigi-client.d.ts#L20

Reply:
Jup indeed. Thx for finding.